docs/advanced-usage/storage-backends/eos/index.md

```diff
@@ -1 +1,36 @@
 # EOS
+
+REANA uses shared filesystem for storing results of your running workflows.
+They may be garbage-collected after a certain period of time. You can use the
+`reana-client download` command to download the results of your workflows or
+copy it to your personal [EOS]((http://information-technology.web.cern.ch/services/eos-service)) space.
+
+To publish your results on EOS you have to add a final step to your workflow
+that would copy the results of interest in the outside filesystem.
+
+First, we have to let the REANA platform know your Kerberos keytab so that the
+writing is authorised. We can do this by [../../access-control/kerberos/index.md](uploading keytab and CERN username).
```

Comment:
Wrong format, should be `[uploading keytab and CERN username](../../access-control/kerberos/index.md)`